AnnoyancesFilter/sections/popups.txt

```diff
@@ -116,6 +116,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+todai.kawai-juku.ac.jp###pushBnr
```

Comment:
Remove this and instead add `||pushcode.jp^$third-party` to push-notifications.txt, this is the cause.

Reply:
ok